pkg/gardenlet/metrics/metrics.go

```diff
@@ -0,0 +1,20 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	runtimemetrics "sigs.k8s.io/controller-runtime/pkg/metrics"
+
+	"github.com/gardener/gardener/pkg/utils/flow"
+)
+
+// Namespace is the metric namespace for the gardenlet.
+const Namespace = "gardenlet"
```

Comment:
This file will be expanded by https://github.com/gardener/gardener/pull/10965, where it has been extracted from to be re-used within this PR. Hence, the structure is already created here.